src/allmydata/node.py

```diff
@@ -233,13 +334,11 @@ class Node(service.MultiService):
     CERTFILE = "node.pem"
     GENERATED_FILES = []
 
-    def __init__(self, config, basedir=u"."):
+    def __init__(self, config):
```

Comment:
I wonder if Tahoe-LAFS has public Python APIs.  If so, I wonder if `allmydata.node.Node` is one of them.  If so, I wonder what compatibility guarantees Tahoe-LAFS means to offer for such APIs.  Is removing a parameter an allowed change without a warning period?

Reply:
I believe that Tahoe makes zero promises about any Python API...@warner?